controllers/modals/editorDecision/form/EditorDecisionWithEmailForm.inc.php

```diff
@@ -99,15 +100,15 @@ function initData($actionLabels = array()) {
 	 * @copydoc Form::readInputData()
 	 */
 	function readInputData() {
-		$this->readUserVars(array('personalMessage', 'selectedAttachments', 'skipEmail', 'selectedLibraryFiles'));
+		$this->readUserVars(['personalMessage', 'selectedAttachments', 'skipEmail', 'selectedLibraryFiles', 'bccReviewers']);
 		parent::readInputData();
 	}
 
 	/**
 	 * @copydoc EditorDecisionForm::fetch()
 	 */
 	function fetch($request, $template = null, $display = false) {
-
+		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_REVIEWER);
```

Comment:
If this is for a single key, could you add a comment at the end of the line with the symbolic key? e.g.:
```
AppLocale::requireComponents(LOCALE_COMPONENT_PKP_REVIEWER); // my.locale.key

Reply:
As I have two keys, I'll ignore.